mindmeld-ui/package-lock.json

```diff
@@ -0,0 +1,13099 @@
+{
```

Comment:
do we need to check in this file?

Reply:
Yup!
https://docs.npmjs.com/files/package-lock.json
Here's the writeup on what that file is for if you're curious.